lib/cog_cmd/ecs/task/list.rb

```diff
@@ -5,51 +5,57 @@ class List < Cog::Command
 
     include CogCmd::Ecs::Helpers
 
-    def run_command
-      client = Aws::ECS::Client.new()
+    attr_reader :family_prefix, :max_results, :status
+    attr_reader :task_definition
```

Comment:
I broke them up to distinguish between options and arguments. Not really necessary, it just made it a bit easier to see if I was missing something.